Finjector.Web/Controllers/ChartsController.cs

```diff
@@ -208,11 +208,49 @@ public async Task<IActionResult> DeleteChart(int id)
         return Ok();
     }
 
-    [HttpGet("detail")]
-    public async Task<IActionResult> Details(string segmentString)
+    // fetch by chart id, when on team/{teamId}/folder/{folderId}/details/{chartId}/{chartString}. requires view permission on chart
+    [HttpGet("details/id")]
+    public async Task<IActionResult> DetailsById([FromQuery] int chartId)
```

Comment:
I'm ok with this but usually we'd just put it in the url like `[HttpGet("details/id/{chartId}")]` and same with the details/string one.